R/conditional-transform.R

```diff
@@ -0,0 +1,215 @@
+#' Conditional transformation
+#'
+#' Create a new variable that has values when specific conditions are met. 
+#' Conditions are specified using a series of formulas: the righthand side is 
+#' the condition that must be true (a `CrunchLogicalExpr`) and the lefthand 
+#' side is where to get the value if the condition on the righthand side is 
+#' true. This is commonly a Crunch variable.
```

Comment:
... but may be a string or numeric value, depending on the type of variable you're constructing.